reference/sbom_naming.md

```diff
@@ -32,7 +32,7 @@ Following [guidance](https://slsa.dev/spec/v1.0/provenance) from SLSA provenance
 | SPDX YAML | artifact-1.0.0.tar.gz | artifact-1.0.0.tar.gz.spdx.yml (or .yaml) |
 | SPDX RDF XML | artifact-1.0.0.tar.gz | Artifact-1.0.0.tar.gz.spdx.rdf (or .rdf.xml)
 
-The .spdx.* and .cdx.* extensions are sourced from the CycloneDX and SPDX guidance on filename extensions for SBOM documents of the corresponding standard and format.
+The .spdx.* and .cdx.* extensions are sourced from the [CycloneDX](https://cyclonedx.org/guides/sbom/extensibility/) and [SPDX](https://spdx.github.io/spdx-ntia-sbom-howto/) guidance on filename extensions for SBOM documents of the corresponding standard and format.
 
 The JSON format files should be considered a mandatory requirement and are always available. The tool support for JSON documents is considered to be better than the other file format options. If other formats are desired, also including the JSON SBOM is optional.
```

Comment:
I'm not sure what "The JSON format files ... are always available" means. Is this referring to availability of the JSON-formatted being in a data store? Is this something sbom-everywhere can enforce?